src/status_im/contexts/wallet/events_test.cljs

```diff
@@ -119,3 +113,51 @@
           effects     (events/update-selected-networks {:db db} props)
           result-fx   (:fx effects)]
       (is (match? result-fx expected-fx)))))
+
+(deftest get-wallet-token-for-all-accounts
+  (testing "get wallet token for all accounts"
```

Comment:
Can we update these test names to have a `-test` suffix? This is the recommended style in #20224